invokeai/app/invocations/bria_text_encoder.py

```diff
@@ -0,0 +1,93 @@
+from typing import Optional
+
+import torch
+from transformers import (
+    T5EncoderModel,
+    T5TokenizerFast,
+)
+
+from invokeai.app.invocations.model import T5EncoderField
+from invokeai.app.invocations.primitives import BaseInvocationOutput, FieldDescriptions, Input, OutputField
+from invokeai.app.services.shared.invocation_context import InvocationContext
+from invokeai.backend.bria.pipeline_bria_controlnet import encode_prompt
+from invokeai.invocation_api import (
+    BaseInvocation,
+    Classification,
+    InputField,
+    LatentsField,
+    invocation,
+    invocation_output,
+)
+
+
+@invocation_output("bria_text_encoder_output")
+class BriaTextEncoderInvocationOutput(BaseInvocationOutput):
+    """Base class for nodes that output a CogView text conditioning tensor."""
+
+    pos_embeds: LatentsField = OutputField(description=FieldDescriptions.cond)
+    neg_embeds: LatentsField = OutputField(description=FieldDescriptions.cond)
+    text_ids: LatentsField = OutputField(description=FieldDescriptions.cond)
```

Comment:
Invoke outputs instances of conditioning fields from text encoders, not latents. These have a semantic, user-facing difference.. See the FLUX implementation, specifically `FluxConditioningField` in `invokeai/app/invocations/fields.py` for an example.